packages/client/utils/relay/toJiraSearchQueryId.ts

```diff
@@ -1,3 +1,4 @@
+/* to deprecate */
 const toJiraSearchQueryId = (meetingId: string) => `jiraSearchQuery:${meetingId}`
 
 export default toJiraSearchQueryId
```

Comment:
i believe this should be removed bc from what i've seen, using `jiraSearchQuery:${meetingId}` as a store id doesn't return anything

Reply:
ahhhh explains it.
I think somewhere in the shuffle we stopped assigning this object a deterministic Id (see https://github.com/ParabolInc/parabol/blob/d457c22f969a1aea8df77ff3fdb10c722dde6745/packages/client/utils/relay/LocalPokerHandler.ts#L32)

Could you use this function in that LocalPokerHandler to generate a deterministic ID for this object?
doing so should fix that case above where you had to build a workaround in handleJiraCreateIssue.ts

Reply:
d'oh. you are totally right, will do!